### ISSUE TITLE
Incorrect error when -loglevel is used

### ISSUE THREAD

**Issue by algr** [DynamoRIO/dynamorio#1938]
So I've got a test failure:
```
drrun -s 90 -quiet -debug -killpg -stderr_mask -- tests/bin/common.decode-bad
```
and I want to rerun it with logging:
```
drrun -s 90 -quiet -debug -loglevel 2 -stderr_mask -- tests/bin/common.decode-bad
```
but it faults whatever option is after -loglevel and its argument: 
```
<Application tests/bin/common.decode-bad (21635). Option parsing error : unknown option -killpg. Terminating>
```
Clearly this is a debug build otherwise -loglevel itself would have been faulted.

**Comment by derekbruening** [comment]
The error messages are odd, but note that you're missing the arg to -stderr_mask.

**Comment by algr** [comment]
Yes I made the command string a bit shorter for the defect report.  But I was seeing the problem on the original.

**Comment by egrimley** [comment]
I've been confused by this sensitivity to the order of options, too, but I think it's expected behaviour. The output from "drrun --help" says:

    ...
       or: drrun [options] [DR options] -- <app and args to run>
       or: drrun [options] [DR options] -c <client> [client options] -- <app and args to run>
       or: drrun [options] [DR options] -t <tool> [tool options] -- <app and args to run>
    
    drrun options (these are distinct from DR runtime options):
    ...

So "-quiet" is s drrun option, and "-loglevel" is a DR runtime option, so they have to appear in that order.
